.github/workflows/bin/generate_cron_workflows.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+
+import sys
+from jinja2 import Environment, FileSystemLoader
+
+
+def validate_inputs(cli_args):
+    try:
+        epoch = cli_args[1]
+    except Exception:
+        raise ValueError('Missing required parameter. '
+                         'Epoch must be provided.')
+
+    return epoch
+
+
+if __name__ == '__main__':
+
+    def write_recipe(jinja):
```

Comment:
I think you should drop the `jinja` argument for this function (and instead just hard-code the template name inline). The reason we would _include_ an argument like that is to swap out with different templates, right? At this point its safe to say that this particular script is very much tailored for building the cron job workflows, and not much else. Speaking of, looks like this function name is some leftover copypasta from your other project related to writing conda recipes - please select a more appropriate name for it, perhaps `write_workflows`?